resqpy/olio/grid_functions.py

```diff
@@ -481,6 +503,7 @@ def actual_pillar_shape(pillar_points, tolerance = 0.001):
     from_top = pp - pp[0]
     xy_drift = np.abs(from_top[:, :, 0]) + np.abs(
         from_top[:, :, 1])  # use Manhattan distance as cheap proxy for true distance
+    print(xy_drift)
```

Comment:
is this also for testing / debugging?

Reply:
Yup, I definitely should have removed all of those random print statements.